app/code/Magento/InventoryApi/etc/acl.xml

```diff
@@ -14,7 +14,8 @@
                 </resource>
                 <resource id="Magento_InventoryApi::stock" title="Inventory Stocks" translate="title" sortOrder="20">
                     <resource id="Magento_InventoryApi::stock_edit" title="Edit Inventory Stocks" translate="title" sortOrder="10" />
-                    <resource id="Magento_InventoryApi::source_delete" title="Delete Inventory Stocks" translate="title" sortOrder="20" />
+                    <resource id="Magento_InventoryApi::stock_delete" title="Delete Inventory Stocks" translate="title" sortOrder="20" />
```

Comment:
why do we introduce independent resource for Stock Edit and Stock Delete? 
why not to have the single ACL resource for both operations? 

Like this one:
```xml
 <resource id="Magento_InventoryApi::stock" title="Manage Inventory Stocks" translate="title" sortOrder="20" />
```

And the same for Source

Reply:
Separate ticket has been created
https://github.com/magento-engcom/magento2/issues/89
need to discuss several things in a scope of ACL refactoring